incubator-tools/language_detection/language_detection.ipynb

```diff
@@ -0,0 +1,535 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    from google.cloud import translate_v3beta1 
</pre><p>Change this to match the other sections.</p><p> </p><pre class="ql-syntax" spellcheck="false">from google.cloud import translate_v3beta1&nbsp;as translate
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Acknowledged.</p>